code/Fluent.php

```diff
@@ -43,9 +43,14 @@ public static function regenerate_routes()
         // Home page route
         $routes[''] = array(
             'Controller' => 'FluentRootURLController',
-            self::config()->query_param => static::default_locale(true),
         );
 
+        // If we do not wish to detect the locale automatically, fix the home page route
+        // to the default locale for this domain.
+        if (!Fluent::config()->detect_locale) {
+            $routes[''][self::config()->query_param] = static::default_locale(true);
+        }
+
         // If default locale doesn't have prefix, replace default route with
```

Comment:
Looks good to me. @tractorcow I think there might be some benefit in having some rudimentary `FunctionalTests` for `Fluent` - not necessarily for _this_ commit, but going forward - what do you think?